charts/argo-cd/templates/argocd-notifications/deployment.yaml

```diff
@@ -2,12 +2,12 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ template "argo-cd.notifications.fullname" . }}
+  name: {{ include "argo-cd.notifications.fullname" . }}
   labels:
     {{- include "argo-cd.labels" (dict "context" . "component" .Values.notifications.name "name" .Values.notifications.name) | nindent 4 }}
 spec:
   strategy:
-    {{- .Values.notifications.updateStrategy | toYaml | nindent 4 }}
+    type: Recreate
```

Comment:
When using something else controller will not kill old AFTER new ones are created. This causes problem with mixed notifications. So the option is now hardcoded.